code/modules/mob/living/silicon/ai/life.dm

```diff
@@ -158,6 +158,8 @@
 			health = 100 - getOxyLoss() - getToxLoss() - getFireLoss() - getBruteLoss()
 		diag_hud_set_status()
 		diag_hud_set_health()
+		ear_damage = null

```

Comment:
@tigercat2000 Strangely enough, as the code currently stands, and you can check in AI, when the value is null, the game automatically makes it a number when they get any ear damage. As the game sits, all AI start with ear damage of null, and nor do they break. when they get a value added.